python/tank/api.py

```diff
@@ -592,10 +594,38 @@ def preview_filesystem_structure(self, entity_type, entity_id, engine=None):
                                                       engine)
         return folders
 
+    def get_site_cache_root(self):
+        """
+        Returns the location of the site cache root for this Tank instance.
+        """
+        # Invoke the global method which is generic and can compute any site
+        # cache root.
+        global get_site_cache_root
+        return get_site_cache_root(self.shotgun.base_url)
 
 ##########################################################################################
 # module methods
 
+
+def get_site_cache_root(base_url):
```

Comment:
Maybe this could be moved into `tank.util` somewhere? Or do we need it at this level? Would be cool if we could keep this file as clean as possible!